transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -123,6 +112,16 @@ static jint netty_epoll_native_eventFd(JNIEnv* env, jclass clazz) {
     return eventFD;
 }
 
+static jint netty_epoll_native_timerFd(JNIEnv* env, jclass clazz) {
+    jint timerFD = timerfd_create(CLOCK_MONOTONIC, TFD_CLOEXEC | TFD_NONBLOCK);
+
+    if (timerFD < 0) {
+        int err = errno;
+        netty_unix_errors_throwChannelExceptionErrorNo(env, "timerfd_create() failed: ", err);
```

Comment:
just a nit: you could merge the two lines. Not need to store `errno` to err first.